packages/node_modules/@webex/plugin-meetings/src/media/index.js

```diff
@@ -15,6 +15,52 @@ import {
  */
 const Media = {};
 
+Media.localTracks = (audio, video, stream) => {
```

Comment:
This calls other functions that return booleans, yet it doesn't return anything. Is its purpose to manipulate data further down? It should probably have a verb or return a boolean (see my note below).